extensions/resource-deployment/src/services/tools/toolBase.ts

```diff
@@ -35,15 +67,88 @@ export abstract class ToolBase implements ITool {
 		return this._statusDescription;
 	}
 
-	public loadInformation(): Promise<void> {
+	protected async getPip3InstallLocation(packageName: string): Promise<string> {
+		const command = `pip3 show ${packageName}`;
```

Comment:
Potential security issue, running a command on what seems to be user input?

Reply:
this is a protected method and all subclasses call this method with literal strings and not based on user input.